pkg/client/client.go

```diff
@@ -593,17 +593,14 @@ func (d *DynatraceClient) ListEntitiesTypes() ([]EntitiesType, RespError) {
 
 	resp, err := d.listPaginated(pathEntitiesTypes, params, "EntityTypeList", addToResult)
 
-	if err.WrappedError != nil {
+	if err != nil {
 		return nil, RespError{
-			WrappedError: err,
-			StatusCode:   resp.StatusCode,
+			Err:        err,
+			StatusCode: resp.StatusCode,
 		}
 	}
 
-	return result, RespError{
```

Comment:
I've replaced several default initialized RespError structs back to nil error returns. 
I think in this case this was wrongly fully initialized. err will be nil at this point (unless I'm missing something)